src/pages/iou/request/step/IOURequestStepMerchant.js

```diff
@@ -75,7 +81,7 @@ function IOURequestStepMerchant({
     const isEditingSplitBill = iouType === CONST.IOU.TYPE.SPLIT && isEditing;
     const {merchant} = ReportUtils.getTransactionDetails(isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction : transaction);
     const isEmptyMerchant = merchant === '' || merchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
-    const isMerchantRequired = _.some(transaction.participants, (participant) => Boolean(participant.isPolicyExpenseChat));
+    const isMerchantRequired = _.some(transaction.participants, (participant) => Boolean(participant.isPolicyExpenseChat)) || ReportUtils.isGroupPolicy(report);
```

Comment:
do we need to look at the participants here at all? I see we also have this method on the reportutils class https://github.com/Expensify/App/blob/07d97efe708f6c609c52ed7b1740fc2a249a4947/src/libs/ReportUtils.ts#L777-L779

so could this just be simplified to check:
```suggestion
    const isMerchantRequired = ReportUtils.isPolicyExpenseChat(report) || ReportUtils.isGroupPolicy(report);
```

cc @mountiny @koko57 since you'd added this here https://github.com/Expensify/App/pull/33844